src/alert.cpp

```diff
@@ -20,7 +20,8 @@ using namespace std;
 map<uint256, CAlert> mapAlerts;
 CCriticalSection cs_mapAlerts;
 
-static const char* pszMainKey = "0486bce1bac0d543f104cbff2bd23680056a3b9ea05e1137d2ff90eeb5e08472eb500322593a2cb06fbf8297d7beb6cd30cb90f98153b5b7cce1493749e41e0284";
+// same as master project key now
+static const char* pszMainKey = "049ac003b3318d9fe28b2830f6a95a2624ce2a69fb0c0c7ac0b513efcc1e93a6a6e8eba84481155dd82f2f1104e0ff62c69d662b0094639b7106abc5d84f948c0a";
```

Comment:
Why did we change this one?

Reply:
> Change Alert **key** to be same as project **master** key.

I suspect that secret key to this key was lost as this feature was never used. I made it same as project master key so Quez and RTM can use it.